### ISSUE TITLE
Happy to Help, Wondering if work is still happening 

### ISSUE THREAD

**Issue by Lighterchu** [OpenHogwarts/hogwarts#84]
Hey Hogwarts, 

I have just finished playing the new hogwarts game, which has made me found this github. With that being said, I have experince to code games for the reason I have self taught myself. Let me know on what coding problems u need done and I can get all fixed up or new features. 

I really hope the game is still being worked on, if not. happy to try and bring it back to life, however I am not a designer only a programmer. 

Hope to hear back and get on the track running. 

If u want to see my previous work, DM me and ill show u what i have created :)

**Comment by xing1357** [comment]
Yeah I am also interested in continuing development. I might create a fork of this soon.
